### DISCUSSION TITLE
Bitwarden releases unified-selfhost option

### DISCUSSION THREAD

**Discussion by l4rm4nd** [dani-garcia/vaultwarden#2961] [General]
The people over at Bitwarden seem to have noticed the demand of Bitwarden for selfhosters. And I guess they also got a bit jealous about the fame vaultwarden got 😉

They released a beta deployment option, featuring a single docker container supporting various database options and ARM devices.

https://bitwarden.com/de-DE/blog/new-deployment-option-for-self-hosting-bitwarden/

Just wanted to spread the news. Let's see how close they can come to vaultwarden.

@dani-garcia again many thanks for developing and maintaining such a great software!

**Comment by dadatuputi** [comment]
What are the differences? I believe one of the advantages of this project is that we have access to many features that are behind a paywall with Bitwarden. Is that still true?

**Reply by BlackDex** [reply]
That is still true as far as i know.
Also, there images do not run om ARMv6, which Vaultwarden images do.
